vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionManager.java

```diff
@@ -147,4 +165,40 @@ private IExhibitionLanguage addExhibitionLanguage(Exhibition exhibition, Map lan
         return exhibitionLanguage;
     }
 
+    @Override
+    public IVSImage storeDefaultImage(byte[] image, String filename, String id) {
```

Comment:
this method should probably be part of the imageservice and be called `storeImage`